db/db_impl/db_impl.h

```diff
@@ -2963,6 +2966,10 @@ class DBImpl : public DB {
   // stores the number of compactions are currently running
   int num_running_compactions_;
 
+  // stores the number of input iterators required for currently running
+  // compactions
```

Comment:
`uint64_t` or `size_t` makes more sense to me since `num_running_compaction_input_iterators_` should not be negative. However, I wanted to follow the convention here since everything else is using `int`